app/views/devise/registrations/start.html.slim

```diff
@@ -1,11 +1,12 @@
 - title t('titles.registrations.start')
 
-h1.h3.my0 = t('headings.create_account')
+h1.h3.my0 = t("headings.create_account_#{service_provider}", sp: @sp_name)
```

Comment:
❤️ the decorator pattern. That's the same idea as the Null Object in that thoughtbot article I linked to above.